Neos.Media.Browser/Configuration/Policy.yaml

```diff
@@ -28,6 +28,10 @@ privilegeTargets:
       label: Allowed to replace asset resources
       matcher: 'method(Neos\Media\Browser\Controller\(Asset|Image)Controller->(replaceAssetResource|updateAssetResource)Action())'
 
+    'Neos.Media.Browser:WorkspaceName':
+      label: Allowed to see workspace owner
+      matcher: 'method(Neos\Media\Browser\Controller\UsageController->relatedNodesAction())'
```

Comment:
We tried it with a normal Editor and he can access the media usage tab, but doesn't see the workspace titles (as expected). Do you maybe have another solution for the privilege?

Reply:
maybe let's add a comment:

```
we only evaluate the WorkspaceName privilegeTarget in code (UsageController) to determine whether we should show workspace names or not. To control access to the module in general, Neos.Media.Browser:AssetUsage is used (so that's why they have the same matcher string).
```